src/test/java/com/zufar/onlinestore/test/config/RestAssertion.java

```diff
@@ -0,0 +1,34 @@
+package com.zufar.onlinestore.test.config;
+
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.springframework.http.HttpStatus;
+
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+import static org.hamcrest.Matchers.*;
+
+public final class RestAssertion {
+
+    public static ValidatableResponse checkStatusCodeInResponse(Response response, int code, String schema) {
```

Comment:
Change `int code` to `HttpStatus httpStatusCode` 

it enables to avoid `HttpStatus.OK.value()`